### DISCUSSION TITLE
Do I need to delete any temporary files?

### DISCUSSION THREAD

**Discussion by PeterPedro** [openai/whisper#1197] [Q&A] [answered]
Hello, I was just wondering since every time I use Whisper there's a download and some are pretty big. I'd like to know if there's any file left over like a temporary file, I would like to know where it is so I can delete it because my PC doesn't have that much space. Also in the case of a crash or something like that the file could be there even if Whisper does usually remove it so it would be nice to know where it is located. Checked my /temp folder and there's nothing there. Using Ubuntu.

**Comment by jongwook** [comment, marked as answer]
The download you're seeing is probably the model files, which can be up to a few gigabytes. You can find the files under `~/.cache/whisper` and delete them to reclaim space. It'll be re-downloaded when you use Whisper next time.
